scripts/aws

```diff
@@ -8,11 +8,14 @@ set -o pipefail
 trap - INT TERM
 
 docker run --rm \
-  -t "$(tty &>/dev/null && echo "-i")" \
-  -e "AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}" \
-  -e "AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}" \
-  -e "AWS_REGION=${AWS_REGION}" \
-  -e "AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION}" \
-  -v "$(pwd):/project" \
-  mesosphere/aws-cli \
-  "$@"
+	-t $(tty &>/dev/null && echo "-i") \
+	-e "AWS_ACCESS_KEY_ID" \
+	-e "AWS_SECRET_ACCESS_KEY" \
+	-e "AWS_REGION" \
+	-e "AWS_DEFAULT_REGION" \
+	-e "AWS_SESSION_TOKEN" \
+	-e "AWS_SECURITY_TOKEN" \
+	-v "$(pwd):/$(pwd)" \
+	-w "$(pwd)" \
```

Comment:
the 'official' mesosphere [aws-cli readme](https://hub.docker.com/r/mesosphere/aws-cli/)  shows `-v "$(pwd):/project"` - their default [workdir is /project](https://hub.docker.com/r/mesosphere/aws-cli/dockerfile)  - this using the pwd inside the container could result in something weird? is doing a static working directory and volume mount better?

Reply:
I'm good either way. Mounting the `pwd` in a one-to-one fashion and overriding the workdir means that the `deploy` script does not have to adapt to running inside the container. Meaning if, for some odd reason, we decided later to not use the mesosphere container we could just swap that for the _real_ aws cli without having to fix the deploy script pathing.

As for any concerns of something weird, no there should be none. It just means we have a more specific use case than the general'ish one they aimed for.

Reply:
Ok, we'll fly with what we have here.